tt_metal/hw/inc/wormhole/dev_mem_map.h

```diff
@@ -93,7 +93,7 @@
 // Stack info
 // Increasing the stack size comes at the expense of less local memory for globals
 #define MEM_BRISC_STACK_SIZE 768
-#define MEM_NCRISC_STACK_SIZE 768
+#define MEM_NCRISC_STACK_SIZE 1040
```

Comment:
did you run BH post commit and check if that needs to be modified?

Reply:
Ran BH CI with Watcher enabled. Things seem okay with the updated stack size... no warnings.